.github/workflows/test-mobile.yml

```diff
@@ -1,45 +1,143 @@
 name: "[Test] Mobile App"
 
 on:
-  push:
-    branches:
-      - main
-      - develop
-      - release
-      - hotfix
+  # push:
+  #   branches:
+  #     - main
+  #     - develop
+  #     - release
+  #     - hotfix
+  schedule:
+    - cron: "0 2/12 * * *"
   pull_request:
   workflow_dispatch:
+    inputs:
+      ref:
+        description: "The branch to run tests on (develop by default)"
+        required: false
+        default: develop
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
same problem as above, we should also check for something like github.event.inputs.ref (not sure of that we need to look into documentation) as well a gtihub.ref

Reply:
Where exactly should this check be done? I've added `ref: ${{ github.event.inputs.ref || 'develop' }}` for the checkout step which I think should be enough